src/profile/helpers/editData.ts

```diff
@@ -127,7 +159,7 @@ function create(
   };
```

Comment:
Comment for line 155: According to my ide the line could be simplified to: Can be simplified to `(profileData as MultiItemProfileNode).primary`

Reply:
Changed. Affected tests and fixed them.